layouts/partials/blocks/newsletter-topper.html

```diff
@@ -34,9 +34,11 @@ <h1 class="my-4 text-4xl font-semibold text-white">
           {{ .Title }}
         </h1>
 
-        <h2 class="mb-4 text-lg font-semibold">
-          <span class="text-white">{{ .Param "white-subhead" }}</span>
-          <span class="text-yellow">{{ .Param "yellow-subhead" }}</span>
+        <h2 class="mb-4 text-lg font-semibold text-white">
+          {{ .Param "white-subhead-first" | strings.TrimSpace }}
+          <span class="text-yellow"
+            >{{ .Param "yellow-subhead" | strings.TrimSpace }}</span
+          >{{ .Param "white-subhead-second" }}
```

Comment:
Maybe instead of having three subheads, write a Tailwind sub-selector so that `<strong>` is in yellow, then do the text with markdown like `Sign Up for **Talk of the Town** for essential news from our State College bureau`.

Reply:
If I understand [the Tailwind docs](https://v3.tailwindcss.com/docs/hover-focus-and-other-states#using-arbitrary-variants) correctly, it would be `[&_strong]:text-yellow` on the parent h2 to target the strong tag.